app/assets/stylesheets/_shared.scss

```diff
@@ -4,7 +4,7 @@
   left: 0;
   right: 0;
   bottom: 0;
-  background-image: url('/assets/header-background.avif');
+  background-image: url('../assets/header-background.avif');
```

Comment:
since it's rails, isn't this enough? `background-image: url('header-background.avif');`